CHANGELOG.md

```diff
@@ -5,63 +5,85 @@ CHANGELOG
-- Add new set of commands to build and manage Custom AMIs.
-- Add new set of commands to list, view and export cluster and image builder logs.
-- Split head node and compute fleet instance roles and add possibility to configure a different instance role
-  for each queue.
+- Add support for pcluster actions (e.g., create-cluster, update-cluster, delete-cluster) through HTTP endpoints
+  with Amazon API Gateway.
+- Add support to use an existing Instance Profile for cluster creation and custom AMI creation.
+- Support restart/reboot for EC2 instance types with 
+  [instance store](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/InstanceStorage.html) (ephemeral drives).
+- Add support to use an existing Private Route53 Hosted Zone when using Slurm as scheduler.
+- Split head node and compute fleet instance roles and add possibility to configure a different instance role for each queue.
 - Add possibility to configure different security groups for each queue.
-- Add support for associating an existing Elastic IP to the head node.
-- Create S3 bucket per region shared with cluster and image if custom bucket isn't specified instead creating bucket
-  per cluster.
-- Encrypt root EBS volumes and shared EBS volumes by default.
-  Note that if the scheduler is AWS Batch, the root volumes of the compute nodes cannot be encrypted by ParallelCluster.
-- Add multiple queues and compute resources support for `pcluster configure` when the scheduler is Slurm.
-- Add prompt for availability zone in `pcluster configure` automated subnets creation.
-- Use different permissions in instance roles based on the scheduler and the node's role in the cluster.
-- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' state
-  in case recurrent failures are encountered when provisioning nodes.
-- Add tag `Name` to every shared storage with the value specified in the shared storage name config.
+- Add support for multiple subnets when using AWS Batch.
+- Allow full control on the name of CloudFormation stacks created by ParallelCluster by removing the `parallelcluster-`
+  prefix.
+- Add multiple queues and compute resources support for pcluster configure when the scheduler is Slurm.
+- Add prompt for availability zone in pcluster configure automated subnets creation.
+- Add configuration HeadNode.Imds.Secured to enable/disable restricted access to Instance Metadata Service (IMDS).
+- Implement scaling protection mechanism with Slurm scheduler: compute fleet is automatically set to 'PROTECTED' 
+  state in case recurrent failures are encountered when provisioning nodes.
 
 **CHANGES**
-- Drop support for SGE and Torque schedulers.
-- Drop support for CentOS8.
+- Remove support for SGE and Torque schedulers.
+- Remove support for CentOS8.
 - Change format and syntax of the configuration file to be used to create the cluster, from ini to YAML.
```

Comment:
Let's also add that a configuration file now represents the settings of a single cluster.